plugins/node/opentelemetry-instrumentation-mongodb/package.json

```
@@ -58,7 +58,7 @@
     "@types/mongodb": "3.6.20",
     "@types/node": "18.6.5",
     "mocha": "7.2.0",
-    "mongodb": "3.6.11",
+    "mongodb": "^6.5.0",
```

We usually pin devDependencies. :slightly_smiling_face: 
```suggestion
    "mongodb": "6.5.0",
```